org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -0,0 +1,171 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+  appName:
+    Type: String
+    Default: 'sso'
+
+  # AWS SSO instance ARN
+  instanceArn:
+    Type: String
+    Default: 'arn:aws:sso:::instance/ssoins-7223577c8c0d796e'
+
+  # Principal ID from Identity Provider's group used by administrators
+  adminGroup:
+    Type: String
+    Default: '906769aa66-4b16d4b3-7c9c-44b7-85e0-adbf41dbf49d'
+
+  # Principal ID from Identity Provider's group used by auditors
+  auditorGroup:
+    Type: String
+    Default: '906769aa66-57ea4266-0644-4316-b5a4-fe0108301d30'
+
+  # Principal ID from Identity Provider's group used by developers
+  developerGroup:
+    Type: String
+    Default: '906769aa66-49d7689b-ae36-472b-bc3d-893753529227'
+
+  # Principal ID from Identity Provider's group used by supporters
+  scienceSupporterGroup:
+    Type: String
+    Default: '906769aa66-5d23a723-54f3-4c08-a67b-311e555f4e85'
+
+SsoAdministrator:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.6/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-admin'
+  StackDescription: 'Full permission role used by Admin group within whole organization'
+  TerminationProtection: false
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: '*'
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref adminGroup
+    permissionSetName: 'Administrator'
+    managedPolicies: [ 'arn:aws:iam::aws:policy/AdministratorAccess' ]
+    sessionDuration: 'PT4H'
+    masterAccountId: !Ref MasterAccount
+
+SsoAdministratorSupporter:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.6/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-admin-supporter'
+  StackDescription: 'Full permission role used by Supporter group within Production organizational units'
+  TerminationProtection: false
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      OrganizationalUnit:
+        - !Ref ScienceDevOU
+        - !Ref ScienceProdOU
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref scienceSupporterGroup
```

Comment:
I don't think this group is granular enough for the two OUs... what if we want to give admin to some accounts within ScienceDevOU, but not all, or all admin to ScienceDevOU but not ScienceProdOU? I think we might need all permutations of a role like this for every account